### PR TITLE
Tracking release `nucypher-core@0.11.0`

### PR DESCRIPTION
**Pull request by piotr-roslaniec**
**Type of PR:**
- Other

**Required reviews:** 
- 1

**What this does:**
> High-level idea of the changes introduced in this PR. 
> List relevant API changes (if any), as well as related PRs and issues.

**Issues fixed/closed:**
> - Fixes #...

**Why it's needed:**
> Explain how this PR fits in the greater context of the NuCypher Network.
> E.g., if this PR address a `nucypher/productdev` issue, let reviewers know!

**Notes for reviewers:**
- Updated `CHANGELOG.md`
- Downstream changes
  - https://github.com/nucypher/nucypher-ts/pull/250
  - https://github.com/nucypher/nucypher/pull/3171

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,8 +3,20 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.11.0] - 2023-08-01
 
-## 0.10.0 - 2023-06-23
+### Added
```
Comment (derekpierre) [inline CHANGELOG.md:8]: Let's put the "Added" (features) section below the "Changed" (incompatible) section - https://github.com/nucypher/nucypher-core/blob/8b1ed8c0f541dc7d921b4e87e794a1cfe999d893/PUBLISHING.md?plain=1#L9

File: nucypher-core-python/stubtest-allowlist.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+nucypher_core.umbral.Parameters.__init__
```
Comment (derekpierre) [inline nucypher-core-python/stubtest-allowlist.txt:1]: Pardon my ignorance, what's this for?

Comment (piotr-roslaniec) [reply nucypher-core-python/stubtest-allowlist.txt:1]: We're using [`mypy.stubtest`](https://github.com/nucypher/nucypher-core/pull/73/files#diff-5e0f5577f0715558b9e0c39adb2d162e51c3f95ea9709aee682aab8a7c9cca65R128) to make sure `*.pyi` files match runtime of bindings generated by `pyo3`

**Review by derekpierre — APPROVED** [review]
🎸

**Pull request merged**